src/webgpu/shader/execution/expression/call/builtin/mix.spec.ts

```diff
@@ -34,8 +49,7 @@ const vec_scalar_cases = (['f32', 'f16'] as const)
           return FP[trait].generateVectorPairScalarToVectorComponentWiseCase(
             FP[trait].sparseVectorRange(dim),
             FP[trait].sparseVectorRange(dim),
-            // NB: Refactor this when adding scalar ranges to FPTrait API
-            trait === 'f32' ? sparseF32Range() : sparseF16Range(),
+            FP[trait].sparseScalarRange(),
             nonConst ? 'unfiltered' : 'finite',
             ...FP.f32.mixIntervals
```

Comment:
This line should be `FP[trait].mixIntervals` and I have comment on the first PR.

Reply:
Done